### PR TITLE
zh-translation: content/zh/docs/ops/configuration/telemetry/envoy-stats

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md [hunk 1]
```diff
@@ -5,41 +5,74 @@ weight: 10
...
+    inclusionPrefixes:
+    - "upstream_rq_retry"
+    - "upstream_cx"
+{{< /text >}}
 
-你可以通过覆盖 Envoy 的默认配置去收集比通常更少的数据。比如可以使用
-`sidecar.istio.io/statsInclusionPrefixes: cluster_manager,listener_manager`
-去收集尽可能少的统计数据。
+{{< tip >}}
+注意：如果您使用 `sidecar.istio.io/statsInclusionPrefixes`， `sidecar.istio.io/statsInclusionRegexps`，和 `sidecar.istio.io/statsInclusionSuffixes`，考虑需要切换到基于 `ProxyConfig` 配置，因此它提供了一个全局默认并且统一的方法去重载网关和 Sidecar 代理。
```
Comment (orangegzx) [inline content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md:77]:
```suggestion
注意：如果您使用 `sidecar.istio.io/statsInclusionPrefixes`，`sidecar.istio.io/statsInclusionRegexps` 和 `sidecar.istio.io/statsInclusionSuffixes`，考虑需要切换到基于 `ProxyConfig` 配置，因此它提供了一个全局默认并且统一的方法去重载网关和 Sidecar 代理。
```

File: content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md [hunk 2]
```diff
@@ -5,41 +5,74 @@ weight: 10
...
+    inclusionPrefixes:
+    - "upstream_rq_retry"
+    - "upstream_cx"
+{{< /text >}}
 
-你可以通过覆盖 Envoy 的默认配置去收集比通常更少的数据。比如可以使用
-`sidecar.istio.io/statsInclusionPrefixes: cluster_manager,listener_manager`
-去收集尽可能少的统计数据。
+{{< tip >}}
+注意：如果您使用 `sidecar.istio.io/statsInclusionPrefixes`，`sidecar.istio.io/statsInclusionRegexps` 和 `sidecar.istio.io/statsInclusionSuffixes`，考虑需要切换到基于 `ProxyConfig` 配置，因此它提供了一个全局默认并且统一的方法去重载网关和 Sidecar 代理。
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md:77]: 网关不翻译。

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**